Exadel.Compreface.AcceptenceTests/Services/FaceDetectionServiceTest.cs

```diff
@@ -88,7 +88,7 @@ public async Task FaceDetectionBase64Async_TakesRequestModel_ReturnsProperRespon
             var response = await _faceDetectionService.FaceDetectionBase64Async(_faceDetectionBase64Request);
 
             // Assert
-            Assert.IsType<FaceDetectionBase64Response>(response);
+            Assert.IsType<FaceDetectionResponse>(response);
```

Comment:
the reason for this change is that it was useless. The declaration of FaceDetectionBase64Response:

```
public class FaceDetectionBase64Response : BaseResult
    {
        public PluginVersions PluginVersions { get; set; }
    }
```
but the compreface postman docs shows that the response will be identical `FaceDetectionResponse` so we don't need duplicate classes. It seems like a convention base64 methods will return same response fields with regular endpoints so for detection we don't need extra base64 response